deepchem/models/atomic_conv.py

```diff
@@ -278,8 +177,6 @@ def __init__(
       layer = prev_layer
       if next_activation is not None:
         layer = Activation(next_activation)(layer)
-      print("size")
-      print(size)
       # layer = Dense(100)(layer)
       layer = Dense(
```

Comment:
One point worth noting here is that this PR (and my earlier PR) both make a major change to the `AtomicConv` architecture by cutting off `AtomicConvScore` and adding a stack of dense layers at the end. This is a little less physically inspired but perhaps more robust in practice.

For us to really have a sense for whether this architecture works, we'll need to run some benchmarks on PDBBind and get numbers. This is a larger project though! Might be a good target for future PRs

Reply:
Yes, this is a big change - once this is merged in I'll start working on the tutorial and do some preliminary testing to see what the scope of a larger benchmark might look like.